packages/pakistan.provinces/paksitan.provinces.svg

```diff
@@ -0,0 +1,57 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  viewBox="0 0 1628 1544"
+  aria-label="Map of Pakistan"
```

Comment:
Could you use the label "Map of Pakistan provinces" or "Map of provinces and territories of Pakistan" please?